asm/istio/options/trustanchor-meshca.yaml

```diff
@@ -0,0 +1,59 @@
+# Copyright 2020 Google LLC
```

Comment:
I don't think this file is supposed to be in the commit, is it?

Reply:
I wanted to add the option file for Mesh CA trustanchor in the same commit. Is that okay?

Reply:
Can you explain a bit how it's related to revision labels? Also do you want it in the same PR, or the same commit? I'd rather the commits be separate even if it's the same PR just to make commit management easier later.

Reply:
I wanted to add it as part of the same PR, not the same commit. This installation option installs the meshCA trust Anchor in the ASM mesh so that  workloads signed by Istiod can communicate with ASM mesh CA workloads.

Reply:
Pushed into two separate commits

Reply:
I can move this out into a separate PR if you think we need to discuss this in more detail

Reply:
If they're in separate commits that's enough for me :)